fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/query/visitors/ExpressionVisitor.java

```diff
@@ -516,6 +516,8 @@ public Expression visitInList(@Nonnull RelationalParser.InListContext ctx) {
         final Expression result;
         if (ctx.preparedStatementParameter() != null) {
             result = visitPreparedStatementParameter(ctx.preparedStatementParameter());
+        } else if (ctx.fullColumnName() != null) {
+            result = visitFullColumnName(ctx.fullColumnName());
```

Comment:
Do you think we should verify that the `result` type is an array?

Reply:
sure, added